spec/api-web-contents-spec.js

```diff
@@ -667,4 +667,14 @@ describe('webContents module', () => {
       w.loadURL(`file://${path.join(__dirname, 'fixtures', 'pages', 'theme-color.html')}`)
     })
   })
+
+  describe('console-message event', () => {
+    it('is triggered with correct log message', (done) => {
+      w.webContents.on('console-message', (e, level, message) => {
```

Comment:
`s/on/once`

It's possible that Chromium spits out deprecation warnings to console, or other console messages that we don't explicitly trigger.  We want to be sure that we only call `done` once 👍

Reply:
Ah, right. I suppose those unexpected logs might precede our 'a', in which case `once` would fail? The latest commit removes the assert which should work for those cases.